upgrade.sh

```diff
@@ -0,0 +1,261 @@
+#!/bin/bash
+#
+# Simple script for upgrading AppScale to one of release versions.
+# Author: AppScale Team <support@appscale.com>
+
+set -e
+
+APPSCALE_REPO="git://github.com/AppScale/appscale.git"
+APPSCALE_TOOLS_REPO="git://github.com/AppScale/appscale-tools.git"
+AGENTS_REPO="git://github.com/AppScale/appscale-agents.git"
+THIRDPARTIES_REPO="git://github.com/AppScale/appscale-thirdparties.git"
+GIT_TAG="last"
+UNIT_TEST="N"
+
+usage() {
+    echo "Usage: ${0} [--tag <git-tag>] [-t]"
+    echo
+    echo "Options:"
+    echo "   --tag <git-tag>    Git tag (e.g.: 3.7.2) to upgrade to."
+    echo "                      Default: '${GIT_TAG}' (use the latest release)."
+    echo "   -t                 Run unit tests"
+    exit 1
+}
+
+version_ge() {
+    test "$(printf '%s\n' "$@" | sort -V | tail -1)" = "$1"
+}
+
+
+echo -n "Checking to make sure you are root..."
+if [ "$(id -u)" != "0" ]; then
+   echo "Failed" 1>&2
+   exit 1
+fi
+echo "Success"
+
+echo -n "Checking to make sure \$HOME is /root..."
+if [ "$HOME" != "/root" ]; then
+   echo "Failed"
+   exit 1
+fi
+echo "Success"
+
+# Let's get the command line argument.
+while [ $# -gt 0 ]; do
+    if [ "${1}" = "--tag" ]; then
+        shift; if [ -z "${1}" ]; then usage; fi
+        GIT_TAG="${1}";
+        shift; continue
+    fi
+    if [ "${1}" = "-t" ]; then
+        UNIT_TEST="Y"
+        shift; continue
+    fi
+    echo
```

Comment:
You could do something like echo -e "\nParameter ...\n" for simplicity if you want to